src/pages/home/userDetail.jsx

```diff
@@ -56,12 +56,12 @@ const UserDetailPage = () => {
     const user = query.get("user");
     setUser(user);
 
-    Data.getUserDetail(user).then((info) => {1
+    Data.getUserDetail(user).then(async (info) => {
+      const timeout = await Data.getTimeout();
       const array = [...info.rituals];
-      console.log("array: ", array)
```

Comment:
did you intend to leave this console output here?

Reply:
Hey, thanks for the review!

This commit is removing the log as it doesn't have any reason to keep it there.